ngraph/frontend/paddlepaddle/src/op/elementwise_ops.cpp

```diff
@@ -15,31 +15,63 @@ NamedOutputs elementwise_ops(const NodeContext& node) {
+        auto axis_is_last = std::make_shared<ngraph::opset6::Equal>(axis, x_rank_sub_1);
+        auto rank_is_equal = std::make_shared<ngraph::opset6::Equal>(x_rank, y_rank);
+        auto not_broadcast = std::make_shared<ngraph::opset6::LogicalOr>(axis_is_last, rank_is_equal);
+        // if broadcast or not
+        auto if_op = std::make_shared<ngraph::op::v8::If>(not_broadcast);
+        auto element_type = y.get_element_type();
+        // not broadcast
+        auto then_x = std::make_shared<ngraph::op::Parameter>(element_type, PartialShape::dynamic());
+        auto then_y = std::make_shared<ngraph::op::Parameter>(element_type, PartialShape::dynamic());
+        auto then_res = std::make_shared<ngraph::op::Result>(then_y);
+        auto then_body =
+            std::make_shared<ngraph::Function>(ngraph::OutputVector{then_res}, ngraph::ParameterVector{then_x, then_y});
+        // broadcast
+        auto else_x = std::make_shared<ngraph::op::Parameter>(element_type, PartialShape::dynamic());
+        auto else_y = std::make_shared<ngraph::op::Parameter>(element_type, PartialShape::dynamic());
+        auto x_shape_else = std::make_shared<ngraph::opset6::ShapeOf>(else_x);
+        auto x_rank_else = std::make_shared<ngraph::opset6::ShapeOf>(x_shape_else);
+        auto y_shape_else = std::make_shared<ngraph::opset6::ShapeOf>(else_y);
+        auto y_rank_else = std::make_shared<ngraph::opset6::ShapeOf>(y_shape_else);
+
+        auto range0 = std::make_shared<ngraph::opset6::Range>(const_0, axis, const_1, ngraph::element::i64);
+        auto y_rank_add_axis = std::make_shared<ngraph::opset6::Add>(y_rank_else, axis);
+        auto y_rank_add_axis_scalar = std::make_shared<ngraph::opset6::Squeeze>(y_rank_add_axis);
+        auto x_rank_scalar = std::make_shared<ngraph::opset6::Squeeze>(x_rank_else);
+        auto range1 = std::make_shared<ngraph::opset6::Range>(y_rank_add_axis_scalar,
+                                                              x_rank_scalar,
+                                                              const_1,
+                                                              ngraph::element::i64);
+        auto indices = std::make_shared<ngraph::opset6::Concat>(ngraph::NodeVector{range0, range1}, 0);
+        auto y_broadcast = std::make_shared<ngraph::opset6::Unsqueeze>(else_y, indices);
+        auto else_res = std::make_shared<ngraph::op::Result>(y_broadcast);
+        auto else_body =
+            std::make_shared<ngraph::Function>(ngraph::OutputVector{else_res}, ngraph::ParameterVector{else_x, else_y});
+        // set input/output/body of if op
+        if_op->set_then_body(then_body);
+        if_op->set_else_body(else_body);
+        if_op->set_input(x, then_x, else_x);
+        if_op->set_input(y, then_y, else_y);
+        auto res_y = if_op->set_output(then_res, else_res);
+
+        return node.default_single_output_mapping({std::make_shared<T>(x, res_y)}, {"Out"});
```

Comment:
minor reminder: better add more comments to illustrate the if-then subgraph.
```
element_wise(X, Y)
Y needed to be broadcast if XXX(explain about when need broadcast)
```

Reply:
Have added the annotation to explain